adhocracy4/comments_async/serializers.py

```diff
@@ -40,7 +40,10 @@ def to_representation(self, instance):
         """
         ret = super().to_representation(instance)
         categories = {}
-        if ret['comment_categories']:
+        if ret['is_blocked']:
+            ret['comment_categories'] = categories
```

Comment:
I don't really get, why you put it here and moved the if and made the other one to an `elif`.